tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/management/BackupService.java

```diff
@@ -31,6 +32,7 @@
 
 @Component
 @RestControllerEndpoint(id = "backups")
+@Profile("standalone")
```

Comment:
can we rather use `@RestControllerEndpoint(id = "tasklist/backups")` to be able to create backups with the single application?